packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -289,7 +290,12 @@ const GlobalHeader = ({ className }) => {
                 size={Button.SIZE.EXTRA_SMALL}
                 variant={Button.VARIANT.PRIMARY}
               >
-                <span>Sign up</span>
+                <span>
+                  {domain === 'www.docs-preview.newrelic.com' ||
+                  domain === 'www.docs.newrelic.com'
+                    ? 'Sign Up'
+                    : 'Start Now'}
```

Comment:
An alternative using `includes` (and moving the logic out of the JSX) might make this a little more succinct:

```jsx
// above the JSX
const inDocsSite = ['https://developer.newrelic.com', 'https://docs-preview.newrelic.com'].includes(siteUrl);

// in the JSX
<span>{isDocSite ? 'Sign Up' : 'Start Now'}</span>
```